superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -432,6 +432,29 @@ export const controls = {
     'to find in the [country] column'),
   },
 
+  freq: {
+    type: 'SelectControl',
+    label: t('Frequency'),
+    default: 'W-MON',
+    freeForm: true,
+    clearable: false,
+    choices: [
+      ['AS', '[AS] Year'],
+      ['52W-MON', '[52W-MON] 52 Weeks (starting Monday)'],
+      ['W-SUN', '[W-SUN] Week (starting Sunday)'],
+      ['W-MON', '[W-MON] Week (starting Monday)'],
+      ['D', '[D] Day'],
+      ['4W-MON', '[4W-MON] 4 Weeks'],
+    ],
+    description: t(
+      'The frequency over which to pivot time. Free-form "pandas" offset alias ' +
```

Comment:
you can use text ticks \`text\` and wrap without concatenation (it inserts spaces for newlines)

Reply:
oh I didn't know ticks did that. neat